### PR TITLE
add tags to swagger

### PR DESCRIPTION
**Pull request by RayBB**
Adds tags for better sorting.
Next step would be to improve the apis themselves based on current docs and testing

Feel free to edit this PR directly as needed to merge

### Screenshot

image

### Stakeholders

@cclauss

### CODE REVIEW

File: static/openapi.json [hunk 1]
```diff
@@ -1,14 +1,60 @@
...
+    }
+  ],
   "paths": {
     "/api/books": {
       "get": {
         "summary": "Read Api Books",
         "operationId": "read_api_books_api_books_get",
+        "tags": [
+          "books"
+        ],
```
Comment (cclauss) [inline static/openapi.json:57]: In #5910, we will need to add this into `openapi_dict` before we write the json or yaml output...
```python
openapi_dict["paths"]["/api/books"]["get"]['tags"] = "XYZ"
```

**Comment by cclauss** [comment]
Let's merge this so that we can all see these changes in the UI.

The problem that we have is that we are modifying a file that is autogenerated in #5910.
We will need to modify our #5910 code to push these changes into the OpenAPI spec document as it is generated.

**Comment by cclauss** [comment]
Please see the two edits that I made in https://github.com/internetarchive/openlibrary/pull/8084/commits/8f7119f9f3875fbfb9f4a72a89feb3f5c4a49ce1 before merging.

File: static/openapi.json (hunk 1 above)
Comment (RayBB) [reply static/openapi.json:57]: I'll think about this some more

**Pull request merged**